challenge_5/python/alexbotello/FindTheDifference.py

```diff
@@ -0,0 +1,21 @@
+from collections import Counter
+
+
+class Solution:
+    def findTheDifference(self, s, t):
+
+        s = dict(Counter(s))
+        t = dict(Counter(t))
+
+        for key in t.keys():
+            if key not in s.keys():
```

Comment:
Okay, yeah I see that now. I'll remove the nested loop and handle the exception instead.